src/integrationTest/java/uk/gov/hmcts/darts/workflow/command/AddAudioMidTierCommand.java

```diff
@@ -118,20 +129,28 @@ public void execute() {
     }
 
     @Override
-    public void executeWithDocker() throws IOException {
+    public void executeWithDocker(Map envVariables) throws IOException {
         File xmlFile = getAudioFile();
 
         try (GenericContainer<?> container = getContainer(xmlFile)) {
             container.start();
 
-            Container.ExecResult result = container.execInContainer(
-                    "sh", "addaudio.sh",
-                    "http://" + ipToGateway + BASE_DOCKER_COMMAND_WEB_CONTEXT,
-                    xmlFile.getName(),
-                    fileToSend.getName());
+            Container.ExecResult result;
+
+            // invoke MTOM or UCF
+            result = container.execInContainer(
+                "sh", "addaudio.sh",
+                "http://" + ipToGateway + BASE_DOCKER_COMMAND_WEB_CONTEXT,
+                xmlFile.getName(),
+                fileToSend.getName());
 
             exitCode = result.getExitCode();
 
+
+            logOutput = result.getStdout();
+            log.info("Container output {} ", result.getStdout());
+            log.info("Container output error {} ", result.getStdout());
```

Comment:
This does not feel correct,

could also reuse the logOutput variable instead of calling the method again

error out and normal out use the same method.

Reply:
Resolved